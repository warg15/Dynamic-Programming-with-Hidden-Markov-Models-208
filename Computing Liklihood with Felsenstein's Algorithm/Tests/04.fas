>Peregrine-falcon
TAGGGGGTCATGTATATGTGCTGGTACCGTGCATCGGCCTAACCTAAGTAGGGTATCATGACACATCGTAGAGGAACGCGATTAGAAAACAATATTAACGTCCGGAGGGGATACTTTCGGATTTAAAAGGGTCTCGGGTAAACTATTTCGTTGGAAGAGTAGATTTACAGCGACTAGGTGGTTAATTATAGATGAAATTT
>Medium-ground-finch
TAAGGCTCCAGGTGTATGTACTAGTACCGCGTATCGGCCTAACCTAACTACACTATTACGACACATCGTAGATGAACGCGAGTGTAGCACAACATTAACAGTGGTAAGGGACATTCTGGGAGTTGAAAGGGTCATGTGTACAGTATTTCATTGGAAGAGTAGATTCATAACGACTAGGTGGTGAATTATAGATGAAGTTT
>Yellow-throated-sandgrouse
TAGAGAGTCATGTGTATGTACTGGTACCGCGCATCGGCCTAATCCAAGCAGGGTATCACGACACATCGTAGATCAACGCGATTGGAAAACAACATTAACGTCCGGAGGGAATACCCTTGGATTTAAAAGGGTTACGGATACACTATTCCGTTGGAAGAGTAGATTTATAGCGACTAGGGGGTTAATTATAGATGAAGTTT
>Chicken
TACGGCTTCGCGCGTATATATTGTTACCGCCCATAGGCCTAACCTAAAAAGGATGTCTCGACACATAGTAGGTGAGCACGAATGGAAAACACGAGTGATATCCGGAGGGAATACCCTTGTATTTTAAAGGGTTAAGGGTATACTATTTCGTCGGAAGAGTAGATTTATAGCAACTAGGTGGTTAATTGTAGTTGAGATTT
>Pekin-duck
TAGGGGATCACGCGTATGTACTGGTACCGCCCATCGGCCTAACCTAAGGAGGATAGCACGACACATAGTAGATGAGCAAAATTGGAAAGCATTATTAACATCCGGAGGGAATACCCTTGTGTTTCAAAGGATTACGGGTACGCTATTTCGTTGGAAGAGTCGTTTTATAGCGACTAGGTAGATAATTATAGGTGAGGTTT
>Zebra-finch
TAAGGGTCCAGGTGTATGTACTGGTACGACGTATCGGCCTAACCTAACTACGGTATTACGACACATCGTAGATGAACGCGAGTGAAACACAACATTAACGGCGGTAGGGAACATTATGGGAGTTGAAAGAGTTATGTGTACAGTATTTGATTGGAAGAGTAGATTCATAACGACTAGGTGGTGAATTATAGATGAAGTTT
>Carmine-bee-eater
TAGGGGGTCATGTGTAGGTACTGGTACCGCGCATCGGCCTAACATAAGTAGGGTATCACGACACATCGTAGACGAACGCGAGTAGAAAACGACATTAACGTCTGGAGGGGATACTTTTGGATTTAAAAGGATCACGGGCACACTATTTCGTTGGAAGAATAGATTTATAGCGACAAGGTGGTTAATTATAGATGAAGTTT
>Red-throated-loon
TAGGGGGTCATGTGAATGTACTGGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAAAATTAACGTCCGGAGGGGCTACTCTTGGATCTAAAAGGGTTACGGGTACGCTATATCGTTGGAAGAGTAGATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTT
>American-flamingo
GAGGGGGTCATGTGTACGTACTGGTACCGCGCATCGGCCTAACCTAAGCAGGGTATCACGACACTTAGTAGATGAACGCGATTGGAAAACAACATTAACGTCCGGAGGGGATACTCTTGGATTTAAAAGGGTTACGAGTACACTATTTCGTTGGAAGAGTAGATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTT
>Chimney-swift
TAGGGGGGCATGTGTATGTACTGGTATCGCGCATAGGCCTAACCTAAGTGTGGTTGCACGACACCTCGTAGATGAACGCGATTGAAAAACAACACCAACATCCGGAGGGGATACTCTTGGATTTATAAGGGTTACGGGTACACTGTTTCGTAGGAAGAGGAGATTTATAACCACTGGGCGGTTAATTATAAATGAAGTTT
>Pigeon
TAGAGGGTCATGTGTATGTACTGGTACCGCGCATAGGCCGAACCTGAGAAGGGTATCATGACACATAGTAGATGAACGCCATTGGAAGGCAACATTAACGTCCCGAGAGGATACTCTTGGATTTAAAAGGGTCACGGGTACACTGTTTCGCTGGAAGAGTAGGTTTATAGCGACTAGGTGGTTAATTATAGATGAAATTT
>White-tailed-tropicbird
TAGAGGGTCGTGTGTATGTACTGATACCGCGCATCGGCCTAACCTAAGTGGGGTAGCACGAGAGATCGTAGATGAACGCGATTAAAAAACAACATTAACGTCCGGGGGGGGTACGCTTGGATTTAAAAGGGTTACGGGCATACTATTTCGTGGGAAGAGCGGATTTATAGCGACTAGGTGGTCAATTATAGATGAAGTTT
>Red-crested-turaco
TAGGGGGTAATGTGTATGTACTGGTACCACGCATCGGCCTAACCTAAGCAGGGTATCACGATACATCGTACATGAACGCGATTAAAAAATAACATTAACGTCCGGTGGGGATACCCTTGGATTTAAAAGGCTTACGGGTACATTATTTTGCTGGAAGAGTGGATTTAAAGCGACTAGTTGGTTAATTATAGATGTAGTGT
>Dalmatian-pelican
TGGGGGGTCACGTGTATGTACTGGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGAACGCGACTGGAAAACAACATTAACGTCCGGAGGGTATACTCTTGGATTTAAAAGGGTTACGGATACACTATATCGTTGGAAGAGTAGATTTATAGCGACCAAGTGGTTAATTATAGATGAAGTTT
>American-crow
TAGGGGTCCAGGTGTGTGTACTGGTATCGCGTATCGGCCTAACCTAAGTGGGGTATTACGACACATCGTAGATGAACGCGAGTGGAACACAACATTAACAGCGGTAGGAGACATTCTGGGAGTTGAAAGGGTCATGTGTACAGTATTTCGTTGGAAGAGTAGATTTATAGCGACTAGGTGGTGAATTATAGATGAAGTTT
>Sunbittern
TAGGTGGTCATGTGTATGTACTGGTAGCGCGCATCGGTCTAACCCACGTAGGGTACCACCACACGTCGTAGATGAACGCGATTGAAAAAAAATATTAACGACCGGAGGGGACACTCTCAAATTTAAAGGGGTTACGGGTACATTATTTCGTTGGAAGAGTAGATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTT
>Annas-hummingbird
TGGGGGGTCATGTGTATGTACTGGTACCGCGCACAAGCCTAACCTGAGTGCGGTAGCCCGGTACATCGTAGATGAACGCGATTGAAACACAACATTAATGTCCGGAGGGGATACTCTTGGATCTAAAAGAGTTACGGGGACACTACCTCGTTGGAAGAGGAGGTTTATAACAACTGCGCGATTAATTATACATGAAGTTT
>Adelie-penguin
TAGGGGGTCATGTGTATGTACTGATACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGAGACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCCAGAGAGGATACTCTTGGATTTAAAAGGGTTACGGGTACACTATATCGTTGGAAGAGTTGATTTATAGTGACTAGGTGGTCAATTATAGCTGAAGTTT
>Northern-fulmar
TAGGAGGTCATTTGTCTGTACTGGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCCGGAGGGGATACTCTTGGATTTGAAAGGGTTACGGGCACACTATATCGTTGGAAGAGTAGATTTATGGCGACTAGGTGGGTAATTACAGATGAAGTTT
>White-tailed-eagle
CAGGGGGTCATCTGTATGTAATGGTACCGCGCGTCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGGACGCGATTGGAGAACAACATTAACGTCCGGAGGGATTACTTTTGGATTTAAAAGGGTTACGGGTACACTATTTCGTTGGAAGTGTAGATTTATAGCGACTATGTGGTTAATTATAGATGAGGTTT
>Speckled-mousebird
TAAAGGGTCATGTGTATGTACTGGTACCGTACATCTGCCTAACCCAAGTAGAGCATCACGACACATCGTAGATGGACACGATTGGAAAACAACATGGGCGTCCAGGGGGGATACTCTTGGATTTAGAACGGTTACGGGTACGCTATTTGGTTAGAAGAGTAGATTCATAGCGACTGGGTGGTTAATTATAGATGAAGCTT
>Emperor-penguin
TAGGGGGTCATGTGTATGTACTGGTACCGCGTATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCCAGAGGGGATACTCTTGGATTTAAAAGGGTTACGGGTACACTATATCGTTGGAAGAGTTGATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTT
>Turkey-vulture
TAGGGGGTCATGTGTATGTACTAGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCGAGAGGGGATACTCTTGGATTTAAAAGGGTTACGGGTACACTATTTCGTTGAAAGTGTAGATTTATAGGGACTAGGTGGTTAATTATAGATGAAGTTT
>Little-egret
TAGGGGGTTACGTGTATGTACTGGTACCGCTCATCGGCCTAGCCTAAGTAGGGTATCACGACACATCGTAGATGAATGGGATTGGAAAACAACATTAACCTCCGGAGGGGATACTCTTGGATTTAAAAGGGTTACGAGTACACTATATCGTTGGAAGAGTAGATTTACAGCGACTAGGTGGTTAATTAGAGATGAAGCTT
>Turkey
TAGGGCTTCGCGCGTATATACTGTTACCGCCCATAGGCCTAACCTAAAAAGGATTTCACGACACATAGTAGGTGGGCACGAATGGAAAACGCTAGTGATATCCGGAGGGAATACCCTTGTATTTTAAAGGGTTACAGGTATACTATTTCGTCGGAAGAGTAGCTTTATAACAACTAGAGGGTTGATTGTAGTTGAGGTTT
>Red-legged-seriema
TAGGGGGTCATGTGTAGGTACTGGTACCGCGCATCTACCTAACCTAAGTAGGGTATCACGACACGTCGTAGATGAACGCAATTGGAAAACAACATTAACGTCCGGAGGGGATACTCTTGGGTTTAAAAGGGTTGCGGGGACACTATTTCGTTGGAAGAGTAGATTTATACCGACTAGGTGGTTAATTATAGATAAAGTTT
>Common-ostrich
CAGAGAGTCATGTGTATGTACTGGTACCGCGCATCAACCTAACCTAAGTAGGGTATCACCACACATAGTAGATGAGCGCGATTGGTAAACACCATTAACGTTCGGAGGGGATACCCTTGGATGCCAAAGGGGTACGCGTACACTATTTCGTTGGCAGAGTAGACTTGTAGCGACTAAGTGGTTAATTATAGATGACGTTT
>Great-crested-grebe
TAGGGGGTCATATGTACATACTGGTACCGCGCATCGGCCTAAGCTAAGTAGGGTATCACGACACATCGTAGATGAATGCGATTAGAAACCAACATCAAAGTCCGGAGGGGATACTCTTGGATTTAACACGGTTGCGGGTACACTATTTCGCTGGAAGAGTAGATTTATAACGACTAGGTGGTTAATTACAGATGAAGTTT
>Budgerigar
TAGAGGATCATGTGTATGGACTCGCACCGCGCATTCGTATAGCCTAAGCGGGGTATCACGACACATCTTTGATGAACGCGATTGAAAAACAACATTAACGCCCGCAGGGGATACTCTTGCATTTAAAAGGGCTACGGGTAGAGTATTTCGTTGGAAGAGTAGATTTAGAGCGCCTAGGTGATTAATTATAAATGGAGTTT
>Great-cormorant
TAGGGGGTAATGTGTATGTAGTGGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACAACACATTGTAGCTGAACGCGATTGGAAAATAACATTAACGTCCAGAGAGGATACTCTTAGATTTAAGAGGGTTACGGGTACACTATATTGTTGGAAGAGTAGATTTATAGCGACTAGGTGGTTAATTATAGAAGAAGTTT
>Bar-tailed-trogon
TAGGGGGTCATGTGTATGTAGTGGTATCGAGAATCAGCCTAACCTGAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAGCATCCGGAGGGGATACGCTTGGAATTAAAAGGGTTGCGGGTACCCTATTTCGTTGGATGATGAAATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTT
>Golden-collared-manakin
TAGGGGGTTAGGTGTATGTCCTGGCACGGCGTATCGGCCTAACCTGAGTAGGGTATTACGACACCCCGTAGATTAACGCGAGTGGGAAACAACATTAATGTCCGTACGAGATACTCTGGGATTTAAAAGGGTTACGGGTACAGTGTTTCGTTGGAAGAGTAGATTTATAGCGATTAGGTGCTAAATTATAGATGAAGTTT
>Downy-woodpecker
TAGGGGGTCATGTGCATGCACTGGTAGTGCGCATCTGCTTAATTTACGTAGGGTATCACGACACATCGTTCATGAACACGATTGGAAAGCAACACTAACGACTGGAAAGGATACTCTTGGATTTAAAAGGGTTATGGGCACACTAATTCATTAGAAGAGTAGATCTACAGTAACTGGGTGGTTAATTATAGATGAAGTTT
>Crested-ibis
TAGCGGGTCATGTGTATGTACTGGTACCGCGCATGGGCCTAACCTAATTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCCGGAGGGAATAATCTTGGATTTAAAAGGGTTACGGGTACACTATATCGTTGGAAGGGTAGATTTATAGCGACTAGGTGGTTAATTATAGATGAAGTTA
>Chuck-wills-widow
TAGGGGGTTATGTGTATGTACTGGTACCGCGCATCGGCCTAACCTAAGTATGGTATCACGACGCATCGTAGATGAACGCGACTGGAAAACAATAGTAACGTCCGGAGGGGATATCCGTGGATTCAAAAGGATTACGGGTAGACTATATCGTTGGAAGACTAGATTTATAGCGACTAGGTGGTTAACTATAGATGAAGGTT
>White-throated-tinamou
TAGACAATCATGTGTATTTACTGATACCGGGCATCAACCTATCCTAAGTAGGATATCACCACACCTAGTGGATGGGCGCGATTGGTGAACACCAATAACGCACAGAGGGGACCCCCTTGGATTACAAAGGAGTACGGGTACACCATTTCGTTAACAGAGTAGATATGTAGCAACTAAGTGGTTAATTATAGATGAAATTT
>Bald-eagle
CAGGGGGTCATCTGTATGTAATGGTACCGCGCGTCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAGATGGACGCGATTGGAGAACAACATTAACGTCCGGAGGGATTACTTTTGGATTTAAAAGGGTTACGGGTACACTATTTCGTTGGAAGTGTAGATTTATAGCGACTATGTGGTTAATTATAGATGAGGTTT
>Hoatzin
TGGGGGGTCATGTGTAAGTACTGGTACCGCGCATCGGCCTAACCTAAGTAGGGTATCACGACACATCGTAAATGAACGCGATTGGAAAACAGCATCAACGTTCAGAGGGGATACTATTGGATTTAAAAGGGTTACGGGTACACTATTTCGCTCGAAGAGTAGATTTATAGCGACTAAGTGATTAATTATAGATGAAGTTT
>MacQueens-bustard
TAGGGGGTCATGTGCCTGTACTGGTACCGCGTATCGGCCTAATCTAAGTAGGGTATCACGAGACATCGTAGATGAACGTGATTGGAAAACACCATTAACGACCGGTGGGGGTACTCTTAGATTTAAAAGGGTTACGGGTACACCACTTCTCTGGTAGAGTCGATTTATAGCGACTAGGTGGTTAATTATAGATAAAGTTT
>Killdeer
TAGGGGGTCATGTGTATGTACTGGTACCGCGCATCTGCCTAACCTAAGTAGGGTATCACGACACATCGTAGACGAACGCGATTGGAAAACAACATTAACGTCCGGAGGGGATACTCATGGATTTAAAAGGGTTACGGGTACACTATTTCATTGGAAGAGCAGATTTATAGCGACTAGGTGGTTAATTTTAGATGAAGTTT
>Rifleman
TAGGGGGTCAGGTGTATTTAGTGGTACCGCGCAACGGCCTAACCGAAGTAGGGTATTACGACACAGCGTAGATGAACGCGAGTGGAAAACAACATTAACGTTCGTTTGGTATACGCTGGGATCTAGTAGGGTCACGGGTACAGTATTTCGTTGGAAGCATAGATTTACAGCGACTAGGTGGTGAATTATTGATGAAGTTT
>Barn-owl
TAGGGGTTCATGTGTATGTACTGGTACTGCGCATCGGCCTAACCTAAGTAGGGTATCACGACAGATCGTAGATGAACGCGATTGGAAAACAACATTAGCGTCAGGAGGGGATACTCTTGGTTTTAAAAGGGTTACGGTTACATTATTTCGTTGGAAAAGTAGATTTATAGCCACTCGGTGCTTAATTATAGACGACGTTT
>Common-cuckoo
TAGGAGGTCATGTGTATTTACTAGTAACGCGCATCGACCTAACCTAAGTAGGGTATCACGGCACTTGGTAGATGAACACGATTGGCAAACCACATTGTCGTCCCGTGGGGATACTCTTGGATTTACTAGGGTTACGGGTATACTATTTCATTGGAAGAGTAGATTTATAGCGACTGGGCAGATAATTATAGATGAAGTCT
>Rhinoceros-hornbill
TACGGGGTCATGTGTATGTAATGGTACCGCGCATTGGCCTAACCTAAGTAGCGTAGCTCGACACGTCGTAAATGAATGCGATTAGAAAACAACATTAACGTCCGGAGGGAATAATCTTGGATTTAAAAAGGTTACGGGTGCACGATTTCGTTGGAAGAGCAGATTTATAGCGAATAGATGGTTAATTATAGATGAAGTCA
>Brown-mesite
TAGAGGGTCATCTCTATGTACTGGTACCGTGTGTAGGCCTATCCTAAGTAGAGTATCCCGACACATCGTCGATGAATGCGGTTGGAAAACAACATTAACGTCCGGAGGGAATACTCTTGGATTTAAAAGGATTATGGGCACACTATTTCGTTGGAAGAGCAGATTTATAGCGACTATGTGGTTAATTATGGATGAAGTTT
>Kea
TAGGGGGTCATGTGTATGGACTGGCACCGCGCATTTGTCTGGCCCAAGTGGGGTATCACGACACATCTTAGATGAACGCGATTGGAAAACAACATTAACGTCCGCAGGGGATACTCTTGGGTTTAATAGGGTTACGGGTAGAGTATTTCGTTGGAAGAGTAGATTTAGTGCGTATGGGTGATTAATTATGAATGGAGTTT
>Cuckoo-roller
TAGAGGGTCATGTGTATGTACTGGTACGGTGCATAGGCCTAACTTAAGTAGGGTATCACGACACATCGTAGATGAACGCGATTGGAAAACAACATTAACGTCCGGATGGGATACTCTTGGGTTTAAAAAGGTTCCGGGTACGCTATTTCGTTGGGAGAGTGGAATTATAGCGACTAGTTGGTCGATTATAGATGAAGTCT
>Grey-crowned-crane
TAGGGGGTCATATATATGTACTGGTACCGCGCATCTGCCTAACCTAAGTAGGGTATCACGACACATTGTAGATGAACGCGACTGGAAAACTACGTTAACGTTCGGAGGGAATACTCTTGGATTTAAAAGGGTTACGGGTGCACTATATAGTCGGAATAGTAGATTGATAGCGACTAGGTGGTTAACTATAGATGAAGTTT
