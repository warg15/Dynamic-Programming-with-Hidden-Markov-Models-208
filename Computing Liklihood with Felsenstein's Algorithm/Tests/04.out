-3526.1686055043992
